network/topics/params/peer_score.go

```diff
@@ -66,9 +65,9 @@ func PeerScoreParams(oneEpoch, msgIDCacheTTL time.Duration, disableColocation bo
 		Topics: make(map[string]*pubsub.TopicScoreParams),
 		// Overall parameters
 		TopicScoreCap: topicScoreCap,
-		DecayInterval: decayInterval,
```

Comment:
wasn't this epochDuration * 100 before?

Reply:
@MatheusFranco99 can you please confirm this is correct?